README.md

```diff
@@ -232,6 +232,10 @@ If you are interested in contributing to this repository, please read up on the
t.com/u/40483304?s=100" width="100px"></a><br /><a title="KwanJunWen's GitHub profile'" href="https://github.com/KwanJunWen"><sub><b>KwanJunWen</b></sub></a></td><td align="center"><a title="MDShields7's GitHub profile'" href="https://github.com/MDShields7"><img alt="MDShields7" src="https://avatars.githubusercontent.com/u/32501364?s=100" width="100px"></a><br /><a title="MDShields7's GitHub profile'" href="https://github.com/MDShields7"><sub><b>MDShields7</b></sub></a></td><td align="center"><a title="AJMcDee's GitHub profile'" href="https://github.com/AJMcDee"><img alt="AJMcDee" src="https://avatars.githubusercontent.com/u/22745495?s=100" width="100px"></a><br /><a title="AJMcDee's GitHub profile'" href="https://github.com/AJMcDee"><sub><b>AJMcDee</b></sub></a></td><td align="center"><a title="tancredosouza's GitHub profile'" href="https://github.com/tancredosouza"><img alt="tancredosouza" src="https://avatars.githubusercontent.com/u/13404051?s=100" width="100px"></a><br /><a title="tancredosouza's GitHub profile'" href="https://github.com/tancredosouza"><sub><b>tancredosouza</b></sub></a></td><td align="center"><a title="Ruchika30's GitHub profile'" href="https://github.com/Ruchika30"><img alt="Ruchika30" src="https://avatars.githubusercontent.com/u/16018506?s=100" width="100px"></a><br /><a title="Ruchika30's GitHub profile'" href="https://github.com/Ruchika30"><sub><b>Ruchika30</b></sub></a></td></tr><tr><td align="center"><a title="anshdhinhgra47's GitHub profile'" href="https://github.com/anshdhinhgra47"><img alt="anshdhinhgra47" src="https://avatars.githubusercontent.com/u/56676059?s=100" width="100px"></a><br /><a title="anshdhinhgra47's GitHub profile'" href="https://github.com/anshdhinhgra47"><sub><b>anshdhinhgra47</b></sub></a></td><td align="center"><a title="gedaiu's GitHub profile'" href="https://github.com/gedaiu"><img alt="gedaiu" src="https://avatars.githubusercontent.com/u/4217037?s=100" width="100px"></a><br /><a title="gedaiu's GitHub profile'" href="https://github.com/gedaiu"><sub><b>gedaiu</b></sub></a></td><td align="center"><a title="i-am-jeetu's GitHub profile'" href="https://github.com/i-am-jeetu"><img alt="i-am-jeetu" src="https://avatars.githubusercontent.com/u/66921514?s=100" width="100px"></a><br /><a title="i-am-jeetu's GitHub profile'" href="https://github.com/i-am-jeetu"><sub><b>i-am-jeetu</b></sub></a></td><td align="center"><a title="Pipo93's GitHub profile'" href="https://github.com/Pipo93"><img alt="Pipo93" src="https://avatars.githubusercontent.com/u/12968697?s=100" width="100px"></a><br /><a title="Pipo93's GitHub profile'" href="https://github.com/Pipo93"><sub><b>Pipo93</b></sub></a></td><td align="center"><a title="jruipinto's GitHub profile'" href="https://github.com/jruipinto"><img alt="jruipinto" src="https://avatars.githubusercontent.com/u/44852805?s=100" width="100px"></a><br /><a title="jruipinto's GitHub profile'" href="https://github.com/jruipinto"><sub><b>jruipinto</b></sub></a></td></tr></table>
```

Comment:
Looks really nice! It seems that semantic-release-bot isn't filtered out, though :-/